app/src/main/res/layout/activity_fragment_container.xml

```diff
@@ -1,7 +1,31 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    android:id="@+id/fragment_container"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    />
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Will all screens use this specific toolbar? Otherwise you could leave it in `activity_viewpager_container`.
In any case, right now both have the same content, one should go.